.github/workflows/test.yml

```diff
@@ -9,6 +9,11 @@ jobs:
       - uses: denoland/setup-deno@004814556e37c54a2f6e31384c9e18e983317366 # v1.1.0
         with:
           deno-version: v1.x
+      - name: Download polkadot
+        run: |
+          docker run -d --entrypoint /bin/sh --rm -it --name tmp parity/polkadot:latest
+          docker cp tmp:/usr/bin/polkadot /usr/local/bin
```

Comment:
The `polkadot` [docker image](https://github.com/paritytech/polkadot/blob/master/scripts/ci/dockerfiles/polkadot/polkadot_builder.Dockerfile#L10) is ubuntu based so it seems to be compatible with the GH workflow runner.

Perhaps there is a better alternative to get the `polkadot` binary 🤔

Reply:
@pepoviola any recommendation? Should we include a runner-compatible Polkadot bin in the Capi repo? Any thoughts would be greatly appreciated.

Reply:
Hey @harrysolovay / @kratico, getting the binary from the `latest` image is a nice way to get the binary. Other way is to download the binary from the `polkadot` repo, as we do in the setup command in zombienet ( https://github.com/paritytech/zombienet/blob/main/src/cli.ts#L120-L141).

Looking forward, we have the task to create a `zombienet action` to publish to the marketplace, then you will be able to just use the action and receive a `json` with the network info (relaychain validators and parachains).
Thanks!

Reply:
Thank you for the input @pepoviola –– will look forward to that action! :)